api/metadata_endpoints.py

```diff
@@ -120,7 +148,7 @@ async def get_collection_metadata(base_url: str, is_self) -> Collection:
             description=f"{ts.standard_name} at {level}m, aggregated over {period} with method '{ts.function}'",
             observedProperty=ObservedProperty(
                 id=f"https://vocab.nerc.ac.uk/standard_name/{ts.standard_name}",
-                label=ts.parameter_name,
+                label=ts.standard_name,
```

Comment:
Can you change all the `label`'s in the `ObservedProperty` to standard name? There is at least one in the CoverageJSON output.